MeetU/MeetU/Content/Meetups.less

```diff
@@ -4,6 +4,7 @@
 
 @meetup-padding-base: 3px;
 @meetup-padding-horizontal: @meetup-padding-base * 2;
+@meetup-padding-vertical: @meetup-padding-base * 4;
```

Comment:
you can't just add padding, because I have designed the position of this button, you should add padding without move this button, in other words, when you add padding, you should reduce margin then the position would keep the same